HOW_TO.md

```diff
@@ -103,23 +104,48 @@ Once the image is built, you can run it locally or push it to an image repositor
 Please note that there may be naming requirements for pushing images to a repository.
 
 **Testing the Docker image**
-Using the same image setup as used when [testing your algorithm](#test), a sample command line to run the image could be:
-```docker run --rm --mount "src=/user/myself,target=/mnt,type=bind" my_algorithm:latest --working_space "/mnt" "/mnt/images"```
+
+using the same image setup as used when testing your algorithm, a sample command line to run the image could be:
+
+```docker run --rm --mount "src=/user/myself,target=/mnt,type=bind" my_algorithm:latest --working_space "/mnt" --metadata "mnt/08f445ef-b8f9-421a-acf1-8b8c206c1bb8_metadata_cleaned.json "/mnt/images"```
```

Comment:
This is **still** using TERRA REF metadata files. These aren't supported for drones